dispatches/models/nuclear_case/Hydrogen_Turbine/hydrogen_turbine.py

```diff
@@ -0,0 +1,151 @@
+#############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
```

Comment:
This isn't a unit model so much as a flowsheet that puts together a Stoichiometric Reactor and a Pressure Changer. I think we probably want to define a new unit model which performs these task that can be instantiated in a flowsheet.